android/app/build.gradle

```diff
@@ -242,9 +235,9 @@ android {
     splits {
         abi {
             reset()
-            enable enableSeparateBuildPerCPUArchitecture
-            universalApk false  // If true, also generate a universal APK
-            include "armeabi-v7a", "arm64-v8a", "x86"
+            enable getEnvOrConfig('ANDROID_ABI_SPLIT').toBoolean()
+            include getEnvOrConfig('ANDROID_ABI_INCLUDE').split(";")
+            universalApk true
```

Comment:
Do we want a universal APK in all scenarios, or just for e2e builds?

Reply:
I think it's useful for testers, don't you think?

Reply:
cc @churik @Serhy what do you think? Is it worth producing and storing an APK that targets all platforms except for Android e2e tests?